acoular/tbeamform.py

```diff
@@ -383,7 +409,7 @@ def result(self, num=2048):
         else:
             fdtype = float32
             idtype = int32
-        w = self._get_weights()
+        self._get_weights()
```

Comment:
this line can be removed completely.

further down we have `weights = self._get_weights()` no need to call the getter twice